.gitignore

```diff
@@ -1,3 +1,4 @@
 _build/**/*
 .DS_Store
 deps/**/*
+.elixir_ls/
```

Comment:
Please remove this line. Ignoring editor specific files can often be solved by a global `.gitignore` :smile:

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Removed